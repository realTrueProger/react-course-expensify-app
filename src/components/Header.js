import React from 'react';
import { NavLink} from 'react-router-dom';

const Header = () => (
    <header>
        <h1>Expensify app</h1>
        <NavLink to="/" activeClassName="is-active" exact={true}>Home page</NavLink>
        <NavLink to="/create" activeClassName="is-active">Create expense</NavLink>
    </header>
);

export default Header;