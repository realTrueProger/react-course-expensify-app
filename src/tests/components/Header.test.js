import React from 'react';
import Header from '../../components/Header';
import { shallow } from 'enzyme';

test('Should render Header component', () => {
    const wrapper = shallow(<Header />);
    expect(wrapper).toMatchSnapshot();

});

