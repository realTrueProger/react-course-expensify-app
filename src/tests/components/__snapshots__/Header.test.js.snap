// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render Header component 1`] = `
<header>
  <h1>
    Expensify app
  </h1>
  <NavLink
    activeClassName="is-active"
    aria-current="page"
    exact={true}
    to="/"
  >
    Home page
  </NavLink>
  <NavLink
    activeClassName="is-active"
    aria-current="page"
    to="/create"
  >
    Create expense
  </NavLink>
  <NavLink
    activeClassName="is-active"
    aria-current="page"
    to="/help"
  >
    Get help
  </NavLink>
</header>
`;
